fishtest/fishtest/views.py

```diff
@@ -512,8 +563,10 @@ def tests_modify(request):
           chunk['pending'] = True
 
     num_games = int(request.POST['num-games'])
-    if num_games > run['args']['num_games'] and not ('sprt' in run['args'] or 'spsa' in run['args']):
-      request.session.flash('Unable to modify number of games in a fixed game test!')
+    if (num_games > run['args']['num_games']
+        and not ('sprt' in run['args'] or 'spsa' in run['args'])):
```

Comment:
If I'm not wrong this should be equivalent and a bit more readable:
```python
if (num_games > run['args']['num_games']
  and 'sprt' not in run['args']
  and 'spsa' not in run['args'])):
```

Reply:
Changed